includes/class-analytics.php

```diff
@@ -101,12 +101,23 @@ public static function get_events() {
 				'on'             => 'scroll',
 				'event_name'     => '100%',
 				'event_value'    => 100,
-				'event_label'    => get_the_title(),
+				'event_label'    => 'success',
```

Comment:
Is that intended? The event label for this article milestone events _is_ supposed to be the article title.

Reply:
Whoops, not intentional. Thanks for pointing out.